rules-reviewed/jboss-eap/websphere/tests/WebSphereMQ.windup.test.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0"?>
+<ruletest xmlns="http://windup.jboss.org/schema/jboss-ruleset">
+    <testDataPath>samples/WebSphereMQ</testDataPath>
+    <rulePath>../WebSphereMQ.windup.xml</rulePath>
+    <ruleset>
+        <metadata></metadata>
+        <rules>
+            <rule>
```

Comment:
I think that tests should have ids, or at least some reference to the rule being tested.